src/java/org/apache/cassandra/service/accord/txn/TxnReference.java

```diff
@@ -194,7 +194,7 @@ public ByteBuffer getFrozenFieldValue(Cell<?> udt)
     {
         UserType userType = (UserType) column.type;
         int field = ByteBufferUtil.getUnsignedShort(path.get(0), 0);
-        return userType.unpack(udt.buffer(), ByteBufferAccessor.instance).get(field);
+        return userType.unpack(udt.buffer()).get(field);
```

Comment:
when i did the last rebase this change got added else the history got too hard to work with; so rolling back